### PR TITLE
Adding danish translation

### PR DESCRIPTION
**Pull request by briis**
## Description
Adding danish translation

## Motivation and Context
As stated above, it adds Danish to the list of translated languages

## How Has This Been Tested
Not tested. But it was created by taking the `en.yaml` file and changing the text to danish. 

## Types of changes

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [x] 🌎 Translation (addition or update a translation)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have tested the change locally.

### CODE REVIEW

File: src/translations/da.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
+{
+    "editor": {
+        "form": {
+            "color_picker": {
+                "values": {
+                    "default": "Stabdard farve"
```
Comment (stalegjelsten) [inline src/translations/da.json:6]: This should probably be "Standard farve"..? :)

Comment (briis) [reply src/translations/da.json]: Thans for catching this. I pushed an update

**Comment by piitaya** [comment]
Hi 🙂 thanks for the contribution. Can you import the file as described in the [readme](https://github.com/piitaya/lovelace-mushroom#maintainer-steps-to-add-a-new-language) ? 🙂

**Comment by briis** [comment]
I have now tested, and all seems correct and is working in the test setup here.

**Pull request merged**